alm/src/config/descriptions.ts

```diff
@@ -0,0 +1,8 @@
+// %t will be replaced by the time -> x minutes/hours/days ago
+export const TRANSACTION_STATUS_DESCRIPTION: { [key: string]: string } = {
```

Comment:
For all messages containing the transaction hash (except one that is not found) it makes sense to display the tx id as a link to Blockscout/Etherscan.

Reply:
Added the links to the explorer https://github.com/poanetwork/tokenbridge/pull/353/commits/dbc7032d65bbb85c1c57889f73fa465d290e1ed7